sys/stdio_uart/stdio_uart.c

```diff
@@ -48,18 +48,20 @@ static uint8_t _rx_buf_mem[STDIO_UART_RX_BUFSIZE];
 isrpipe_t stdio_uart_isrpipe = ISRPIPE_INIT(_rx_buf_mem);
 #endif
 
+static void _isrpipe_write_one_wrapper(void *arg, uint8_t value)
+{
+    isrpipe_write_one(arg, value);
+}
+
 void stdio_init(void)
 {
-    uart_rx_cb_t cb;
-    void *arg;
+    uart_rx_cb_t cb = NULL;
+    void *arg = NULL;
 
-#ifdef MODULE_STDIO_UART_RX
-    cb = (uart_rx_cb_t) isrpipe_write_one;
-    arg = &stdio_uart_isrpipe;
-#else
-    cb = NULL;
-    arg = NULL;
-#endif
+    if (IS_USED(MODULE_STDIO_UART_RX)) {
```

Comment:
```suggestion
#ifdef MODULE_STDIO_UART_RX
```
is needed because `stdio_uart_isrpipe` is not always defined